src/app/pages/submission-pages/data-page/data-page.component.ts

```diff
@@ -127,7 +119,7 @@ export class DataPageComponent implements OnInit {
   }
 
   onSaveContinue() {
-    this.router.navigate(['/submission/samples'])
+
```

Comment:
Should not this navigate somewhere?

Reply:
Sorry Karel I resolved this by mistake. No its gonna be covered in

https://trello.com/c/RtEk7HsD/28-metadata-submission-page